static/explorer.scss

```diff
@@ -347,7 +344,7 @@ pre.content.wrap * {
 
 .compiler-list {
     display: block;
-    overflow: auto;
+    height: auto;
```

Comment:
have you tested this? this doesn't work for long lists with limited space 
![image](https://user-images.githubusercontent.com/442630/131645597-e0abbadf-a7f9-437f-891e-19075451944c.png)

Reply:
I did, but not this exact configuration. Grat catch! :)

Reply:
Fixed